### ISSUE TITLE
Error while installation on Ubuntu 14.04

### ISSUE THREAD

**Issue by snehitgajjar** [chef-cookbooks/mysql#460]
Reference[ Issue #373](https://github.com/chef-cookbooks/mysql/issues/373)

**Comment by kcorupe** [comment]
Same issue, specifying either mysql version 5.5 or 5.6 it tries to install package versions that don't exist in apt.

```
       STDERR: E: Version '5.5.49-0ubuntu0.14.04.1' for 'mysql-client-5.5' was not found
       ---- End output of apt-get -q -y install mysql-client-5.5=5.5.49-0ubuntu0.14.04.1 ----
```

```
       STDERR: E: Version '5.6.30-0ubuntu0.14.04.1' for 'mysql-client-5.6' was not found
       ---- End output of apt-get -q -y install mysql-client-5.6=5.6.30-0ubuntu0.14.04.1 ----
```

Work around is to specify the package_version directly.

```
mysql_client 'default' do
  version '5.6'
  package_version '5.6.31-0ubuntu0.14.04.2'
  action :create
end
```

**Comment by someara** [comment]
Fixed up in 8.0.2

**Comment by webm0nk3y** [comment]
I am getting this in 8.03 on ubuntu 14.04.2

```
==> default: ---- Begin output of apt-get -q -y install mysql-server-5.6=5.6.31-0ubuntu0.14.04.2 ----
==> default: STDOUT: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: Some packages could not be installed. This may mean that you have
==> default: requested an impossible situation or if you are using the unstable
==> default: distribution that some required packages have not yet been created
==> default: or been moved out of Incoming.
==> default: The following information may help to resolve the situation:
==> default:
==> default: The following packages have unmet dependencies:
==> default:  mysql-server-5.6 : Depends: mysql-server-core-5.6 (= 5.6.31-0ubuntu0.14.04.2) but 5.6.33-0ubuntu0.14.04.1 is to be installed
==> default:                     Recommends: libhtml-template-perl but it is not going to be installed
==> default: STDERR: E: Unable to correct problems, you have held broken packages.
==> default: ---- End output of apt-get -q -y install mysql-server-5.6=5.6.31-0ubuntu0.14.04.2 ----
```
